.github/workflows/client-build.yml

```
@@ -112,8 +112,7 @@ jobs:
 
   browser-resend:
     # skip to get test suite green, not sure why failing.
-    if: ${{ false }}
-    name: Test Browser Resend
+    name: Test Browser Realtime
```

Should this be:
```suggestion
    name: Test Browser Resend
```

Also comment above should be removed.